test/jdk/javax/swing/text/html/parser/HtmlTagParserTest.java

```diff
@@ -0,0 +1,82 @@
+import java.awt.Dimension;
```

Comment:
where's the GPL header on this test ?

Reply:
Added the header and moved the test to the HTML folder instead since I believe the process involves classes beyond just the Parser. For example, I found that the tags and unknown tags are located in the HTML class. I can move it back if it seems more appropriate since the issue requests changes to the Parser's doc only.

Reply:
Shall the test name be `HtmlScriptTagParserTest.java`? The test is for testing handling the `<script>` tag only.